src/utils/DialogMatches.ts

```diff
@@ -11,6 +11,7 @@ export const DialogMatches = {
     HelloDialogMatch2: /hi/i,
     HelpDialogMatch: /help/i,
     HeroCardDialogMatch: /hero card/i,
+    PopUpSignInDialogMatch: /sign in/i,
```

Comment:
Let's make the space in the regex optional so if I typed "sign in" or "signin", it would trigger the dialog.

Reply:
Done.